packages/redux/src/bags/actionTypes.ts

```diff
@@ -14,6 +14,22 @@ export const ADD_BAG_ITEM_REQUEST =
 export const ADD_BAG_ITEM_SUCCESS =
   '@farfetch/blackout-redux/ADD_BAG_ITEM_SUCCESS';
 
+/**
+ * Action type dispatched when the set promocodes from bag request fails.
```

Comment:
I think this reads better:

```suggestion
 * Action type dispatched when the set bag promocodes request fails.
```

What do you think? If you agree, then do the same for the remaining actions, please.

Reply:
Changed 👍